opal/mca/btl/tcp/btl_tcp.h

```diff
@@ -351,5 +351,19 @@ mca_btl_tcp_dump(struct mca_btl_base_module_t* btl,
   */
 int mca_btl_tcp_ft_event(int state);
 
+/*
+ * A blocking send on a non-blocking socket. Used to send the small
```

Comment:
socking is non-blocking while send. I have already updated recv method comments 

+ * A blocking recv for both blocking and non-blocking socket. 
 + * Used to receive the small amount of connection information 
 + * that identifies the endpoints
 + * 
 + * when the socket is blocking (the caller introduces timeout) 
 + * which happens during initial handshake otherwise socket is 
 + * non-blocking most of the time.